torch/_dynamo/eval_frame.py

```diff
@@ -827,6 +827,24 @@ def result_capturing_wrapper(*graph_inputs):
     assert out_guards is not None, "Failed to produce guards during tracing"
     assert fake_mode is not None
 
+    if tracing_mode == "symbolic":
```

Comment:
I think it will always be a fake tensor received to the export backend? So probably no need for check here

Reply:
ok, let's just use fake_mode than.